src/Microsoft.AspNetCore.SignalR.Core/Internal/DefaultHubDispatcher.cs

```diff
@@ -142,26 +142,18 @@ public override IReadOnlyList<Type> GetParameterTypes(string methodName)
         private async Task ProcessInvocation(HubConnectionContext connection,
             HubMethodInvocationMessage hubMethodInvocationMessage, bool isStreamedInvocation)
         {
-            try
+            // If an unexpected exception occurs then we want to kill the entire connection
+            // by ending the processing loop
```

Comment:
Is this comment still true?

Reply:
Yes we’re just changing how his happens.